### PR TITLE
Replace hex and RGB values

### PR DESCRIPTION
**Pull request by abhijitb**

### CODE REVIEW

**Review by Yashita101 — CHANGES_REQUESTED** [review]

**Review by arunshenoy99 — COMMENTED** [review]
Let's hold off on reviewing/making further changes to this until the crazy domains branch gets merged to trunk, we might just have to consolidate a few things for CD as well and ensure that we are on the correct variable names. Not removing the WIP until then.

EDIT: The CD branch is now merged into trunk.

File: src/OnboardingSPA/components/CheckboxTemplate/CheckboxItem/stylesheet.scss
Comment (abhijitb) [reply src/OnboardingSPA/components/CheckboxTemplate/CheckboxItem/stylesheet.scss]: Requested changes are done.

Comment (arunshenoy99) [reply src/OnboardingSPA/components/CheckboxTemplate/CheckboxItem/stylesheet.scss]: Hey guys, quick question here, if you are referring to the theme naming conventions
`background->base` and `foreground->contrast` I don't see how `light->base` and `dark->contrast` applies, I see  `nfd-onboarding-contrast` used even for background colors and similarly `nfd-onboarding-base` for text colors/borders as well, should we change places where we have used the words `background/bg` or `foreground/fg` to this convention and leave actual color descriptions like `light-gray` that define a shade accurately as it is?

Comment (abhijitb) [reply src/OnboardingSPA/components/CheckboxTemplate/CheckboxItem/stylesheet.scss]: Requested changes are done.

File: src/OnboardingSPA/components/MiniPreview/stylesheet.scss
Comment (Yashita101) [inline src/OnboardingSPA/components/MiniPreview/stylesheet.scss:2]: can the brand variable be used directly instead of `$browser-light`, like how it's done in other files?

File: src/OnboardingSPA/pages/Steps/GetStarted/SiteTypeSetup/SecondarySite/stylesheet.scss
Comment (abhijitb) [reply src/OnboardingSPA/pages/Steps/GetStarted/SiteTypeSetup/SecondarySite/stylesheet.scss:10]: Colors can be used directly ... If they are used at more than one place then assigning a variable makes sense

File: src/OnboardingSPA/components/MiniPreview/stylesheet.scss
Comment (abhijitb) [reply src/OnboardingSPA/components/MiniPreview/stylesheet.scss]: done .. using the brand variable now

File: src/OnboardingSPA/components/CheckboxTemplate/CheckboxListSkeleton/stylesheet.scss
Comment (abhijitb) [reply src/OnboardingSPA/components/CheckboxTemplate/CheckboxListSkeleton/stylesheet.scss]: added to the branding variables

File: src/OnboardingSPA/components/Drawer/stylesheet.scss
Comment (abhijitb) [reply src/OnboardingSPA/components/Drawer/stylesheet.scss:465]: done .. using `--nfd-onboarding-black` in place of `$main-color-dark`

File: src/OnboardingSPA/components/Tooltip/stylesheet.scss
Comment (abhijitb) [reply src/OnboardingSPA/components/Tooltip/stylesheet.scss]: done

File: src/OnboardingSPA/pages/Steps/GetStarted/Welcome/stylesheet.scss
Comment (abhijitb) [reply src/OnboardingSPA/pages/Steps/GetStarted/Welcome/stylesheet.scss]: done ... moved `#ccc` to a branding variable and used it everywhere else

File: src/OnboardingSPA/components/Drawer/stylesheet.scss
Comment (abhijitb) [reply src/OnboardingSPA/components/Drawer/stylesheet.scss:418]: So currently there is no variable per brand for the color black ... it is not need since if its black across brands it can be defined in a common location ... that is why its defined in the _interface.css and should be used that way only

**Review by Yashita101 — APPROVED** [review]

**Pull request merged**
